app/src/main/java/com/group80/uoftinder/CreateAccountView.java

```diff
@@ -0,0 +1,264 @@
+package com.group80.uoftinder;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.android.material.chip.Chip;
+import com.google.android.material.chip.ChipGroup;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.group80.uoftinder.create_account_use_case.CreateAccountPresenter;
+import com.group80.uoftinder.entities.User;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+
+public class CreateAccountView extends AppCompatActivity {
+
+    private ArrayList<HashSet<Integer>> answers = new ArrayList<>();
+    private final UserAccountController control = new UserAccountController();
+    private final CreateAccountPresenter proceed = new CreateAccountPresenter();
```

Comment:
I believe this presenter is currently not in use (and the source file is commented out). Do we still need it?